gravitee-am-reporter/gravitee-am-reporter-kafka/src/test/java/io/gravitee/am/reporter/kafka/JUnitConfiguration.java

```diff
@@ -35,31 +40,56 @@
 public class JUnitConfiguration implements ApplicationListener<ContextClosedEvent> {
 
     private KafkaContainer kafkaContainer;
+    private SchemaRegistryContainer schemaRegistryContainer;
+    private static final Network NETWORK = Network.newNetwork();
+    private static final String CONFLUENT_PLATFORM_VERSION = "7.6.1";
 
     @Bean
-    public KafkaContainer kafkaContainer(){
-        this.kafkaContainer = new KafkaContainer(
-            DockerImageName.parse("confluentinc/cp-kafka:6.1.1"));
+    public KafkaContainer kafkaContainer() {
+        kafkaContainer = new KafkaContainer(
+                DockerImageName.parse("confluentinc/cp-kafka:" + CONFLUENT_PLATFORM_VERSION))
```

Comment:
question: would it make sense to do something like
`cofluentVersion = Sytem.getProperty("confluentVersion", CONFLUENT_PLATFORM_VERSION)` instead? It would allow us to run tests with a different version without recompiling, like what we're doing with repositories tests. Not sure if/how useful it would be.

Reply:
There will be only one version tested. This is not the same case as repository.